### DISCUSSION TITLE
I'm getting some SSL errors when searching for manga

### DISCUSSION THREAD

**Discussion by HenryTrang** [getmango/Mango#303] [Q&A]
All the plugins I tried that gives me this error:

`Search failed. Error: Error: SSL_connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed`

Does anybody know why this is happening?

**Comment by hkalexling** [comment]
Hey which plugin are you using? I tried the MangaDex plugin and it works fine. Also it could be that the SSL certificate has expired on the target website. Does it work when you try to access it from your browser directly?

**Reply by HenryTrang** [reply]
I tried it with all of the available plugins. I tried it with MangaDex and then I tested MangaDex's SSL certs, they seem to be working fine.
